src/dai.rs

```diff
@@ -82,6 +94,17 @@ impl std::ops::Sub for Amount {
     }
 }
 
+impl From<Erc20> for Amount {
+    fn from(_erc20: Erc20) -> Self {
+        // TODO: inner BigInt is private in comit lib - what is the best way to convert?
```

Comment:
@D4nte @luckysori would you have any suggestions for this?

Reply:
Note: I fixed it by re-constucting the BigInt from bytes for the moment.

Reply:
You could use `comit::asset::Erc20` as the inner of our type.

Reply:
Yeah! Will record it as follow up TODO, because it touches quite a lot of unrelated code parts.